File: src/com/ptby/dynamicreturntypeplugin/config/FunctionCallConfigList.java
package com.ptby.dynamicreturntypeplugin.config;

import java.util.ArrayList;

public class FunctionCallConfigList extends ArrayList<FunctionCallConfig> {
    public FunctionCallConfigList( FunctionCallConfig... functionCallConfigs ) {
        for ( FunctionCallConfig functionCallConfig : functionCallConfigs ) {
            add( functionCallConfig );
        }
    }
}

File: src/main/kotlin/com/ptby/dynamicreturntypeplugin/config/FunctionCallConfigList.kt
package com.ptby.dynamicreturntypeplugin.config

import java.util.*

class FunctionCallConfigList(vararg functionCallConfigs: FunctionCallConfigKt) : ArrayList<FunctionCallConfigKt>() {
    init {
        for (functionCallConfig in functionCallConfigs) {
            add(functionCallConfig)
        }
    }
}
